specs-go/config.go

```diff
@@ -469,6 +462,23 @@ type WindowsNetworkResources struct {
 	EgressBandwidth *uint64 `json:"egressBandwidth,omitempty"`
 }
 
+// LinuxSeccomp represents syscall restrictions
+type LinuxSeccomp struct {
+	DefaultAction LinuxSeccompAction `json:"defaultAction"`
+	// Architectures is kept to maintain backward compatibility for projects
+	// that already use the old seccomp profile.
+	Architectures []Arch          `json:"architectures,omitempty"`
+	ArchMap       []Architecture  `json:"archMap,omitempty"`
```

Comment:
`archMap` is a strange name for a property that is not an object/map.  Maybe call the new setting `architectures2`?  Or take advantage of the pre-1.0 lack of backwards compat (SemVer wording for pre-releases in [§9][1]) and just use `architectures` for the new field.  I'm not a maintainer, but my personal preference is for the latter, since then we don't have to define how to handle configs that set both properties simultaneously.

[1]: http://semver.org/spec/v2.0.0.html#spec-item-9